devel/hs-transformers/Portfile

```diff
@@ -0,0 +1,20 @@
+PortSystem          1.0
+PortGroup           haskell 1.0
+
+haskell.setup       transformers 0.4.3.0
```

Comment:
That probably means a newer version was required. Check with `ghc-pkg list | grep transformers`. If it lists more than one version, this needs the same treatment as `hs-mtl`.